kitsune/search/v2/es7_utils.py

```diff
@@ -17,15 +17,16 @@ def _insert_custom_filters(locale, filter_list, char=False):
     on the value of the `char` argument.
     """
 
-    def mapping_func(filter):
+    def mapping_func(index_filter):
+        index, filter = index_filter
```

Comment:
since this is related to search in general, I would avoid to use the `index` word as it causes confusion with the search index. Anything like `ctr`, `positional_index` or anything that makes it clearer that we are not talking about the search index should do. Maybe a plain comment is the simplest. Nice workaround

Reply:
and maybe replace index_filter with something else? filter_args?

Reply:
I went with `index_filter` as that was the approach [used in the PEP which removed tuple parameter unpacking](https://www.python.org/dev/peps/pep-3113/#no-loss-of-abilities-if-removed) (which would have been useful here):

```python
def fxn(a, b_c, d):
    b, c = b_c
    pass
```

I'll add a `_tuple` suffix to make things clearer.